deploy/helm/templates/rbac.yaml

```diff
@@ -89,6 +89,14 @@ rules:
       - watch
       - create
       - update
+  - apiGroups:
+      - coordination.k8s.io
```

Comment:
Do we need that permission? I see that by default controller-runtime writes to ConfigMap not Lease. Is it somehow configurable?

Reply:
controller-runtime uses `configmapleases` (both configmap and lease) by default. They introduced it to migrate from configmap to lease. I think they will remove configmap support in the future.

Reply:
I tried giving events create permission, then a new error shows up. I'm not sure why this is showing no kind registered for lease.

```
E0324 18:05:48.829011       1 event.go:329] Could not construct reference to: '&v1.Lease{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"starboard-operator", GenerateName:"", Namespace:"starboard-operator", SelfLink:"/apis/coordination.k8s.io/v1/namespaces/starboard-operator/leases/starboard-operator", UID:"7b828f2b-235a-4034-911a-d3bc6c1dc231", ResourceVersion:"379025", Generation:0, CreationTimestamp:v1.Time{Time:time.Time{wall:0x0, ext:63752121432, loc:(*time.Location)(0x22f3080)}}, DeletionTimestamp:(*v1.Time)(nil), DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry{v1.ManagedFieldsEntry{Manager:"starboard-operator", Operation:"Update", APIVersion:"coordination.k8s.io/v1", Time:(*v1.Time)(0xc000117758), FieldsType:"FieldsV1", FieldsV1:(*v1.FieldsV1)(0xc000117770)}}}, Spec:v1.LeaseSpec{HolderIdentity:(*string)(nil), LeaseDurationSeconds:(*int32)(nil), AcquireTime:(*v1.MicroTime)(nil), RenewTime:(*v1.MicroTime)(nil), LeaseTransitions:(*int32)(nil)}}' due to: 'no kind is registered for the type v1.Lease in scheme "pkg/runtime/scheme.go:101"'. Will not report event: 'Normal' 'LeaderElection' 'starboard-starboard-operator-57cd876b88-2jn2j_5a1380bc-b308-49a8-b64d-fef54a645a32 became leader'
```

Reply:
I think we just need to add coordination/v1 GV to `runtime.Scheme` returned by the `starboard.NewScheme()` factory. I tried that and got rid of all errors:

![fix](https://user-images.githubusercontent.com/1322923/112382304-fde86080-8ceb-11eb-9a7b-e054c503cc3e.png)